docs/coordinator_guide.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+displayTitle: Firestorm Coordinator Guide
+title: Firestorm Coordinator Guide
+description: Firestorm Coordinator Guide
+---
+
+# Firestorm Coordinator Guide
```

Comment:
We can some description about `dynamicEnable` and how to use DelegateShuffleManager.

Reply:
There will be two extra PRs for coor and server